docs/streams/developer-guide/processor-api.html

```diff
@@ -159,6 +163,100 @@ <h2><a class="toc-backref" href="#id1">Overview</a><a class="headerlink" href="#
                     arrived records for stateful processing needs like aggregations and joins. For more information, see the <a class="reference internal" href="#streams-developer-guide-state-store"><span class="std std-ref">state stores</span></a> documentation.</p>
             </div>
         </div>
+        <div class="section" id="unit-testing-processors">
+            <h2>
+                <a class="toc-backref" href="#id9">Unit Testing Processors</a>
+                <a class="headerlink" href="#connecting-processors-and-state-stores"
+                   title="Permalink to this headline"></a>
+            </h2>
+            <p>
+                Once you have a processor implementation, you will want to test it.
+                Because the <code>Processor</code> forwards its results to the context rather than returning them,
+                Unit testing requires a mocked context capable of capturing forwarded data for inspection.
+                For this reason, we provide a <code>MockProcessorContext</code> in our <code>test-utils</code>module.
+            </p>
+            <p>
+                Instructions on importing the <code>test-utils</code> module are <a href="testing.html">here</a>.
+            </p>
+            <b>Construction</b>
+            <p>
+            To begin with, instantiate your processor and initialize it with the mock context:
+            <pre>
+final Processor processorUnderTest = ...;
+final MockProcessorContext context = new MockProcessorContext();
+processorUnderTest.init(context);
+                </pre>
+            If you need to pass configuration to your processor or set the default serdes, you can create the mock with
+            config:
+            <pre>
+final Properties config = new Properties();
+config.put(StreamsConfig.APPLICATION_ID_CONFIG, "unit-test");
+config.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "");
+config.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass());
+config.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Long().getClass());
+config.put("some.other.config", "some config value");
+final MockProcessorContext context = new MockProcessorContext(config);
+                </pre>
+            </p>
+            <b>Captured data</b>
+            <p>
+                The mock will capture any values that your processor forwards. You can make assertions on them:
+            <pre>
+processorUnderTest.process("key", "value");
+
+final Iterator&lt;CapturedForward&gt; forwarded = context.forwarded().iterator();
+assertEquals(forwarded.next().kv(), new KeyValue<>(..., ...));
```

Comment:
huh. That was an oversight on my part, but it actually renders correctly, and IDEA didn't warn about it. Maybe since `<>` isn't valid HTML, it just interprets it as text...

Anyway, I'm escaping it now.